integration/kubernetes/runtimeclass_workloads/pod-quota-deployment.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   selector:
     matchLabels:
       purpose: quota-demo
-  replicas: 3
+  replicas: 2
```

Comment:
Like @grahamwhaley, I'm still not clear on this change. It might "work" with `2`, but why did we choose `3` initially then? What did we break?

Maybe @egernst or @mcastelino have thoughts?

Reply:
@GabyCT there does not seem to be a co-relation between the bug and this fix. Can you expand? Were we failing deployment initially?

Reply:
@jodh-intel and @mcastelino before this PR, we never check if kubernetes tests were leaving pod information behind. Until recently, we found out that this test was leaving pod information that is why as part of this PR, I am including to check that after running all the kubernetes tests. If we leave it like it is with 3 replicas then we will have pod info at /var/lib/vc/sbs. The deployment of the replicas is done successfully and even it says that when we remove the deployment is donde successfully, however, after cleaning everything the pod info is at /var/lib/vc/sbs.

Reply:
@jodh-intel , we choose 3 before because the test was trying to run 3 containers but the resource quota was only 2 so the test verified that it was not possible to run 3 containers and that only 2 were running